citesphere/src/main/java/edu/asu/diging/citesphere/core/authority/impl/ConceptpowerImporter.java

```diff
@@ -0,0 +1,156 @@
+package edu.asu.diging.citesphere.core.authority.impl;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.PostConstruct;
+
+import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.client.LaxRedirectStrategy;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import edu.asu.diging.citesphere.core.authority.IImportedAuthority;
+import edu.asu.diging.citesphere.core.exceptions.AuthorityServiceConnectionException;
+import edu.asu.diging.citesphere.model.authority.IAuthorityEntry;
+import edu.asu.diging.citesphere.model.authority.impl.AuthorityEntry;
+import edu.asu.diging.citesphere.web.user.AuthoritySearchResult;
+
+@Component
+@PropertySource(value = "classpath:/config.properties")
+public class ConceptpowerImporter extends BaseAuthorityImporter {
+
+    private final String ID = "authority.importer.conceptpower";
+
+    @Value("${_conceptpower_authority_search_keyword}")
+    private String conceptpowerSearchKeyword;
+
+    @Value("${_conceptpower_url}")
+    private String conceptpowerURL;
+
+    private RestTemplate restTemplate;
+
+    @PostConstruct
+    private void postConstruct() {
+        restTemplate = new RestTemplate();
+    }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
+     * edu.asu.diging.citesphere.authority.impl.AuthorityImporter#isResponsible(java
+     * .lang.String)
+     */
+    @Override
+    public boolean isResponsible(String source) {
+
+        if (CONCEPTPOWER.equals(source)) {
+            return true;
```

Comment:
this should ideally take a uri and decide if it's responsible similar to the viaf importer, since `retrieveAuthorityData` would then try to get the authority data form the uri (which we might implement later).